04-spatial-operations.Rmd

```diff
@@ -77,7 +77,7 @@ that we can also subset simple features using the `filter()` function, e.g. with
 africa_wgs = world[world$continent == "Africa", ]
 ```
 
-To further set up the input data, we will reproject the data to the coordinate reference system (CRS) 32630, its EPSG code (explained in Chapter 6):
+To further setup the input data, we will reproject the data to the coordinate reference system (CRS) 32630, its EPSG code (explained in Chapter 6):
```

Comment:
Ahh in this context the single word version does sound a bit strange to me. I think `setup` is a noun. Therefore `set up` would be fine by I suggest `prepare` would be better still. Cheers for updating the PR!

Reply:
Done that.